docs/api/v2/node.md

```diff
@@ -40,11 +40,15 @@ proc subscribe*(node: WakuNode, request: FilterRequest, handler: ContentFilterHa
   ##
   ## Status: Implemented.
 
-proc unsubscribe*(w: WakuNode, topic: Topic) =
-  ## Unsubscribe from a topic.
+proc unsubscribe*(w: WakuNode, topic: Topic, handler: TopicHandler) {.async.} =
```

Comment:
We started calling WakuNode node instead of w to be more explicit and avoid ambiguity with other things that could be called w. It hasn't been done uniform for all calls yet, but would be good to fix now at least for the unsubscribe calls.

Reply:
Fixed in 4979718